packages/playground-examples/copy/ja/TypeScript/Language Extensions/Enums.ts

```diff
@@ -0,0 +1,81 @@
+// EnumsはTypeScriptを用いてJavaScriptで
+// 固定値の集合を簡単に扱うための機能です。
+
+// デフォルトでは、enumは0から始まり
+// オプション1つにつき1ずつ増加していく数値です。
+// これは値が重要でないときに便利です。
+
+enum CompassDirection {
+  North,
+  East,
+  South,
+  West,
+}
+
+// enumオプションに注釈を付けると、値を設定できます。
+// 注釈をつけなければ、値のインクリメントは設定された値を引き継いで行われます。
+
+enum StatusCodes {
+  OK = 200,
+  BadRequest = 400,
+  Unauthorized,
+  PaymentRequired,
+  Forbidden,
+  NotFound,
+}
+
+// enumはEnumName.Valueで参照できます。
```

Comment:
[imo] Enumname.Valueで伝わるかが悩ましい、そういった予約語のように見えてしまう気がします。
```suggestion
// 各EnumはEnum名.値名で参照できます。
```